components/Questionnaire/Notification/index.vue

```diff
@@ -125,6 +133,27 @@ export default {
       } else {
         return false
       }
+    },
+    showPotencyAction () {
+      if (this.potencyVillages.includes('Perikanan') || this.potencyVillages.includes('Pertanian')) {
+        return true
+      } else {
+        return false
+      }
+    },
```

Comment:
include() methods already returning true or false, so we can simply write 
`return this.potencyVillages.includes('Perikanan') || this.potencyVillages.includes('Pertanian')`
#cmiiw 
same with two function below

Reply:
Refer to [mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes). `include` will returning true or false. I think it's readable and simple if I write as your suggestion. Thanks 👍🏽